1-Canvas-API-Token.md

```diff
@@ -28,7 +28,8 @@
 > Never share your token with anyone. If you think your token may have been exposed (for example, by accidentally posting it to GitHub), delete your token from Canvas right away. Instructions for creating and deleting access tokens as a student are [available on the Canvas Guides](https://community.canvaslms.com/docs/DOC-16005-42121018197).
 
 ## Start the server
-1. Type `npm start` into terminal in `backend` directory. If your server is already running, kill it first: `control + c`, then start the server.
+1. Navigate to the `backend` directory and run `npm install` to install the backend dependencies.
```

Comment:
This is where the installation instructions are in the main readme: https://github.com/UBC-LA-Hackathon/student-dashboard#%EF%B8%8F-getting-started

But ya I think having it here again is a good plan!

Reply:
Updated!